app/core/ts/utils/io/Importer.ts

```diff
@@ -2,11 +2,14 @@ import {SAVED} from "../Config";
 
 import {XMLReader} from "./xml/XMLReader";
 import {ResolveVersionConflict} from "./VersionConflictResolver";
-import {DigitalCircuitDesigner} from "digital/models/DigitalCircuitDesigner";
+import {DigitalCircuitController} from "site/digital/controllers/DigitalCircuitController";
```

Comment:
things in app/ should **never** have a dependency on things in site/

We should have LoadCircuit take in a Camera as a parameter